.prettierrc.json

```diff
@@ -1,6 +1,9 @@
 {
 	"$schema": "http://json.schemastore.org/prettierrc",
-	"overrides": [{ "files": ".nvmrc", "options": { "parser": "yaml" } }],
+	"overrides": [
+		{ "files": ".*rc", "options": { "parser": "json" } },
+		{ "files": ".nvmrc", "options": { "parser": "yaml" } }
+	],
```

Comment:
[Question] @michaelfaith I'm confused: when I run `npx prettier . --write` without these changes, the `.all-contributorsrc` file _does_ get parsed as JSON and written to properly. So why would this fix things?